### PR TITLE
[PRM-1422] - Updated calculation controller to use principal charge reference and penalty category

### PR DESCRIPTION
**Pull request by LewisMitchell**
Update calculation page caption as well due to lib upgrade

### CODE REVIEW

File: app/controllers/CalculationController.scala [hunk 1]
```diff
@@ -49,16 +49,17 @@ class CalculationController @Inject()(viewLPP: CalculationLPPView,
...
-      getOldPenaltyData(penaltyId, isAdditional)
-    }
+    logger.debug(s"[CalculationController][onPageLoad] - Making call to old endpoint")
+    getOldPenaltyData(penaltyId, isAdditional)
+  }
+
+  def onPageLoadForNewAPI(penaltyId: String, penaltyCategory: String): Action[AnyContent] = authorise.async { implicit request =>
+    logger.debug(s"[CalculationController][onPageLoadForNewAPI] - Making call to new endpoint")
+    val penaltyCategoryEnum = LPPPenaltyCategoryEnum.find(penaltyCategory).get
+    getPenaltyDetailsFromNewAPI(penaltyId, penaltyCategoryEnum)
```
Comment (BackRowOla) [inline app/controllers/CalculationController.scala:62]: Could you change penaltyId to principalChargeReference to keep things consistent in the method sig and in 'getPenaltyDetailsFromNewAPI'

**Review by BackRowOla — CHANGES_REQUESTED** [review]
Looks good, just 1 minor comment

File: app/controllers/CalculationController.scala (hunk 1 above)
Comment (LewisMitchell) [reply app/controllers/CalculationController.scala:62]: Good spot, I'm surprised that even worked in the first place 🤔

Comment (BackRowOla) [reply app/controllers/CalculationController.scala:62]: Yeah I know, i think as long as the type is correct it doesn't really care too much what the name is

**Pull request merged**